src/test/kotlin/com/sourcegraph/common/NotificationGroupsTest.kt

```diff
@@ -0,0 +1,56 @@
+package com.sourcegraph.config
+
+import com.intellij.notification.NotificationGroupManager
+import com.sourcegraph.common.NotificationGroups
+import junit.framework.TestCase.assertEquals
+import kotlin.reflect.full.memberProperties
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@Suppress("RedundantLambdaOrAnonymousFunction", "ConstantConditionIf")
+@RunWith(JUnit4::class)
+class NotificationGroupsTest {
+
+  @Test
+  fun `all IDs are valid`() {
+
+    val objectIds =
+        NotificationGroups::class
+            .memberProperties
+            .filter { it.getter.call() is String }
+            .map { it.getter.call() as String }
+            .toSet()
+    // This is just here so we get nice inspection hints
```

Comment:
so this whole test class is for manual testing only? 🤔

Reply:
No the test actually checks that all stated ids exist. BUT by stating it with literal strings you _also_ get a type error directly in newer JetBrains versions where it says the ID isn't defined.